scipy/signal/tests/test_windows.py

```diff
@@ -92,6 +93,50 @@ def test_basic(self):
                          6.0e-05])
 
 
+class TestTaylor(object):
+
+    def test_basic(self):
+        assert_allclose(windows.taylor(1, 2, -15), 1.0)
+        assert_allclose(windows.taylor(5, 2, -15),
+                        np.array([0.75803341, 0.90757699, 1.0,
+                                  0.90757699, 0.75803341]))
+        assert_allclose(windows.taylor(6, 2, -15),
+                        np.array([0.7504082, 0.86624416, 0.98208011,
+                                  0.98208011, 0.86624416, 0.7504082]))
+
+    def test_correctness(self):
+        """This test ensures the correctness of the implemented Taylor
+        Windowing function. A Taylor Window of 1024 points is created, its FFT
+        is taken, and the Peak Sidelobe Level (PSLL) and 3dB and 18dB bandwidth
+        are found and checked.
+
+        A publication from Sandia National Laboratories was used as reference
+        for the correctness values [1]_.
+
+        References
+        -----
+        .. [1] Armin Doerry, "Catalog of Window Taper Functions for
+               Sidelobe Control", 2017.
+               https://www.researchgate.net/profile/Armin_Doerry/publication/316281181_Catalog_of_Window_Taper_Functions_for_Sidelobe_Control/links/58f92cb2a6fdccb121c9d54d/Catalog-of-Window-Taper-Functions-for-Sidelobe-Control.pdf
+        """
```

Comment:
I'd be surprised if `lint` didn't require a `# noqa: E501` here

Reply:
It looks like they are OK with it!